### DISCUSSION TITLE
Inadequete signposting for useFilters requiring defaultColumn for columns with no filters

### DISCUSSION THREAD

**Discussion by tyteen4a03** [TanStack/table#2699] [Ideas]
Title. After an hour of renderer errors I had to dig deep into the example to finally realise I need defaultColumn.

Alternatively, `Filter` should be opt-in by default. If it is not specified then `column.render()` should render nothing.

**Comment by tannerlinsley** [comment]
Thanks for noting this issue. To fix it, a breaking change is required and thus we will slate the fix for v8 and, for sake of triage, close this issue.

Thanks for your patience!
